js/app/src/components/Video/Video.svelte

```diff
@@ -14,8 +14,8 @@
 	export let elem_id: string = "";
 	export let elem_classes: Array<string> = [];
 	export let visible: boolean = true;
-	export let value: FileData | null | string = null;
-	let old_value: FileData | null | string = null;
+	export let value: Array<FileData> | null = null;
```

Comment:
value is specifically a [FileData, FileData] I believe, more accurate than just an Array of arbitrary size. Also applies to type hints below